src/Plugins/SortAnimation/SortAnimation.js

```diff
@@ -16,6 +16,8 @@ export const defaultOptions = {
 };
 
 /**
+ * (Added in: v1.0.0-beta.11)
```

Comment:
Hi @zjffun ,

SortAnimation plugin was added in beta.10 ([changelog](https://github.com/Shopify/draggable/blob/master/CHANGELOG.md#v100-beta10---2020-06-18)).
All other changes are correct, thanks a lot 🍺

Reply:
Oops, thanks for correcting